network/commons/common.go

```diff
@@ -57,10 +64,16 @@ func GetTopicBaseName(topicName string) string {
 
 // CommitteeSubnet returns the subnet for the given committee
 func CommitteeSubnet(cid spectypes.CommitteeID) uint64 {
-	subnet := new(big.Int).Mod(new(big.Int).SetBytes(cid[:]), new(big.Int).SetUint64(SubnetsCount))
+	subnet := new(big.Int).Mod(new(big.Int).SetBytes(cid[:]), bigIntSubnetsCount)
 	return subnet.Uint64()
 }
 
+// SetCommitteeSubnet returns the subnet for the given committee, it doesn't allocate memory but uses the passed in big.Int
+func SetCommitteeSubnet(bigInst *big.Int, cid spectypes.CommitteeID) uint64 {
+	bigInst.SetBytes(cid[:])
+	return bigInst.Mod(bigInst, bigIntSubnetsCount).Uint64()
+}
```

Comment:
i think it would make more sense if it just set the value and not returned it

the caller can just call Uint64() and it would emphasize the big.Int is being modified

that's less convenient, but it's an optimized func so it's OK imo